src/__tests__/commands/utilities/server.test.ts

```
@@ -0,0 +1,419 @@
+/**
+ * Test Suite: Server Command
+ *
+ * Comprehensive tests for the server command module.
+ * Tests cover command structure, guild information retrieval, embed formatting,
+ * timestamp handling, and error scenarios.
+ */
+
+import { describe, it, expect, beforeEach, vi } from 'vitest';
+import { SlashCommandBuilder, EmbedBuilder, ChatInputCommandInteraction, Guild } from 'discord.js';
```

ESLint

[error] 'SlashCommandBuilder' is defined but never used.

(@typescript-eslint/no-unused-vars)